code/modules/projectiles/ammunition/boxes.dm

```diff
@@ -62,14 +62,17 @@
 		overlays += I
 
 /obj/item/ammo_magazine/shotholder/attack_hand(mob/user)
-	if((user.a_intent == I_HURT) && (stored_ammo.len))
```

Comment:
Now won't trying to pick it up off the floor withdraw a casing instead? Please readd the intent check or add some other way to pick it up off the floor.